docs/new-architecture-appendix.md

```diff
@@ -292,7 +307,69 @@ node node_modules/react-native/scripts/generate-specs-cli.js \
 
 In the above example, the code-gen script will generate several files: `MyLibSpecs.h` and `MyLibSpecs-generated.mm`, as well as a handful of `.h` and `.cpp` files, all located in the `ios` directory.
 
-## V. Note on Existing Apps
+## V. Codegen Naming Conventions
+
+### Native Components
+
+Every native component spec file name must follow the following format:
```

Comment:
```suggestion
The spec file name of every native component must follow the following format:
```
This reads better as the subject is actually the spec file name. It's better to put it at the beginning of the sentence.

Reply:
I also think that this is duplicated info. We have the same information in the guides to create Native Components and Native Modules.
Having the same info in multiple places will make it harder to keep them updated.